src/events/logs/messageDelete.js

```diff
@@ -34,6 +34,15 @@ export default async (message, client) => {
 	).entries.first()
 	if (!fetchedLog) return
 
+	// On vérifie si le thread contient plus qu'un message
+	// car il y a le message d'origine à sa création
+	// Si oui alors on archive, sinon on supprime
+	if (message.hasThread) {
+		const thread = await message.thread.fetch()
+		if (thread.messageCount > 1 && !thread.archived) thread.setArchived(true)
+		else thread.delete()
```

Comment:
Si le thread est archivé, qu'importe s'il est vide ou non il sera supprimé. Il faut ignorer si le thread est déjà archivé
```suggestion
		if (!thread.archived) {
				if (thread.messageCount > 1) thread.setArchived(true)
				else thread.delete()
		}
```